addons/mail/static/src/chatter/web_portal/chatter.js

```diff
@@ -105,6 +105,9 @@ export class Chatter extends Component {
 
     _onMounted() {
         this.changeThread(this.props.threadModel, this.props.threadId);
+        this.store.env.services["bus_service"].addChannel(this.state.thread.busChannel);
+        const currentUser = this.store.self;
+        this.store.env.services["bus_service"].addChannel(`res_${currentUser.type}_${currentUser.id}`);
```

Comment:
Why do we do this on the chatter? Current user is not supposed to change, so I would expect this subscribe to only be done once (actually I think it's already done in an override of `_build_bus_channel_list`. Also, `currentUser` is misleading as it could actually be a guest in which case the channel you are adding looks malformed (`res_guest`) is not a valid model?